newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardAppearanceDialog.js

```diff
@@ -0,0 +1,330 @@
+          }}
+          actions={[
+            <FlatButton
+              label={<Trans>Cancel</Trans>}
+              disabled={isLoading}
+              onClick={onClose}
+              key={'cancel'}
+            />,
+            <RaisedButton
+              primary
+              label={<Trans>Save</Trans>}
+              disabled={isLoading}
+              onClick={() => onSaveSettings(i18n)}
+              key={'save'}
+            />,
+          ]}
+        >
+          <Line>
+            <TextField
+              fullWidth
+              floatingLabelText={<Trans>Score column title</Trans>}
+              maxLength={20}
+              errorText={scoreTitleError}
+              value={scoreTitle}
+              onChange={(e, newTitle) => {
+                if (!!scoreTitleError && !!newTitle) setScoreTitleError(null);
+                setScoreTitle(newTitle);
+              }}
+            />
+          </Line>
+          <Text size="title">
+            <Trans>Score formatting</Trans>
+          </Text>
+          <Column>
+            <Line>
+              <SelectField
+                fullWidth
+                value={scoreType}
+                floatingLabelText={<Trans>Score display</Trans>}
+                onChange={(e, i, newValue) =>
+                  // $FlowIgnore
+                  setScoreType(newValue)
+                }
+              >
+                <SelectOption
+                  key={'custom'}
+                  value={'custom'}
+                  primaryText={t`Custom display`}
+                />
+                <SelectOption
+                  key={'time'}
+                  value={'time'}
+                  primaryText={t`Display as time`}
+                />
+              </SelectField>
+            </Line>
+            <ResponsiveLineStackLayout noColumnMargin>
+              <Column noMargin expand>
+                <Line noMargin>
+                  <Text size="body2">
+                    <Trans>Settings</Trans>
+                  </Text>
+                </Line>
+                {scoreType === 'custom' ? (
+                  <>
+                    <Line expand>
+                      <TextField
+                        fullWidth
+                        floatingLabelText={<Trans>Prefix</Trans>}
+                        maxLength={10}
+                        value={prefix}
+                        hintText={t`Ex: €`}
+                        onChange={(e, newValue) => {
+                          setPrefix(newValue);
+                        }}
+                      />
+                    </Line>
+                    <Line expand>
+                      <TextField
+                        fullWidth
+                        floatingLabelText={<Trans>Suffix</Trans>}
+                        maxLength={10}
+                        value={suffix}
+                        hintText={t`Ex: gold`}
+                        onChange={(e, newValue) => {
+                          setSuffix(newValue);
```

Comment:
From your storybook example, it looks like the user has to enter the space in the suffix.
Should the space be always included by default meaning they will not forget it? (I can't think of a case where a user would not want a space between their score & suffix?)

Reply:
We might want to display something like `15$`, so requiring the space is maybe good - though a bit weird I agree.

Reply:
yup, as discussed, even if a bit weird, it's better 👍